lua/cmake/utils.lua

```diff
@@ -89,7 +90,17 @@ function utils.get_current_target(parameters)
     return nil, nil
   end
 
-  local target_dir = vim.fn.fnamemodify(target, ':h')
+  local target_dir
+  local run_dir = parameters['runDir']
+  if run_dir == nil then
+    target_dir = vim.fn.fnamemodify(target, ':h')
+  else
+    local target_dir = Path:new(run_dir)
+    if not target_dir:is_absolute() then
+      target_dir = Path:new(build_dir .. run_dir):normalize(vim.loop.cwd())
```

Comment:
Could you check if `normalize` here is really required?

Reply:
It isn't needed. It seems to work with just `target_dir = build_dir .. run_dir`, but I have only tested a few cases. I do not know if there are any edge cases where this wouldn't work. I will make that change for now though until we run into an issue with that.